### PR TITLE
docs website search POC

### PR DESCRIPTION
**Pull request by anna-lach**
Search based on elasticlunr

#37

### CODE REVIEW

File: docs/website/src/styles/categories/components.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.wrapper {
+  column-gap: 32px;
```
Comment (jpzwarte) [inline docs/website/src/styles/categories/components.scss:2]: I believe we can now just use `gap: 0 32px;` (32px horizontal gap)

File: docs/website/src/styles/categories/components/button.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+h3 {
+  margin-bottom: 16px;
+}
+
+.ds-buttons-wrapper {
+  column-gap: 16px;
```
Comment (jpzwarte) [inline docs/website/src/styles/categories/components/button.scss:6]: Same here

File: docs/website/src/styles/categories/components/button.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  margin: 16px 0;
+  row-gap: 16px;
```
Comment (jpzwarte) [inline docs/website/src/styles/categories/components/button.scss:10]: So replace `column-gap` and `row-gap` with just `gap: 16px;`

File: packages/core/src/button-bar/button-bar.ts [hunk 4]
```diff
@@ -19,6 +19,7 @@ export type ButtonBarAlign = 'start' | 'center' | 'end' | 'space-between';
  */
 export class ButtonBar extends LitElement {
   /** @private */
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```
Comment (jpzwarte) [inline packages/core/src/button-bar/button-bar.ts:22]: You can remove this; there's a timing issue where if you haven't built `button-bar.scss.ts`, then it doesn't know the type of `styles`.

**Review by jpzwarte — APPROVED** [review]

File: packages/core/src/button-bar/button-bar.ts (hunk 4 above)
Comment (anna-lach) [reply packages/core/src/button-bar/button-bar.ts:22]: right, I just forgot about it :)

**Pull request merged**